coa-helm/templates/reverse-proxy.yaml

```diff
@@ -1,4 +1,6 @@
-{{ include "terra-batch-libchart.reverse-proxy-deploy" (list . "coa-helm.reverse-proxy-deploy") }}
+{{- if .Values.cromwell.coaEnabled }}
```

Comment:
I don't think we want this here -- probably conditionally enable each proxy path in the conf file itself (in the base chart).

Reply:
Ah right -- updated the nginx conf in `terra-batch-libchart/templates/_reverse-proxy.tpl` -- I _think_ I included everything in the conditional related to `cromwell`, but I think you would know much better than I haha -- let me know if anything else should be moved out of the conditional